toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -325,3 +330,30 @@ func enableOrDisableServices(services imagecustomizerapi.Services, baseConfigPat
 
 	return nil
 }
+
+func loadOrUnloadModules(modules imagecustomizerapi.Modules, imageChroot *safechroot.Chroot) error {
+	var err error
+
+	for _, module := range modules.Load {
+		logger.Log.Infof("Loading module (%s)", module.Name)
+		moduleFileName := module.Name + ".conf"
+		moduleFilePath := filepath.Join(imageChroot.RootDir(), "/etc/modules-load.d/", moduleFileName)
+		err = file.Write(module.Name, moduleFilePath)
+		if err != nil {
+			return fmt.Errorf("failed to write module load configuration: %w", err)
```

Comment:
Might be worth adding the `module.Name` to the error string.

Same thing below.